controllers/C_Document.class.php

```diff
@@ -660,7 +660,7 @@ function queue_action_process() {
 			
 			//validate that the pod exists
 			$d = new Document($doc_id);
-			$sql = "SELECT pid from patient_data where pubpid = '" . $file['patient_id'] . "'";
```

Comment:
THis might require a major conversion step/script.   Changes to update the documents table at least.  (for people that have existing sites)  just FYI